evm/box/package.json

```diff
@@ -13,6 +13,7 @@
     "format": "prettier --write \"**/*\"",
     "migrate:dev": "npx truffle migrate --reset --network cldev",
     "migrate:live": "npx truffle migrate --network live",
+    "setup": "echo \"No setup required for @chainlink/box\" && exit 0",
```

Comment:
I realized the `exit 0` is a noop, since echo will always return 0 in this case. Also `&&` would cause a non 0 exit code anyway if it failed, this is my bad for writing it everywhere